examples/iam_policy.json

```diff
@@ -192,14 +192,41 @@
         {
             "Sid": "list",
             "Action": [
-                "s3:ListBucket"
+                "s3:ListBucket",
+                "s3:ListAllMyBuckets",
```

Comment:
I don't know why this has changed - I don't think I changed the code on `s3` bucket listing stuff

Reply:
yeah, better change that back

Reply:
@isichei I think `ListAllMyBuckets` and `GetBucketLocation` are supposed to be in the `iam_policy.json`?

Reply:
Yeah agree - doesn't look like I updated the IAM Policy with the latest push. Have added an issue #15 .